wordpress-mobile/WordPress-iOS
# [SIWA] Add flow for 2FA accounts

Fixes #12477 

WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/201
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/158

This imports the WPKit & WPAuth changes to support signing in with SIWA 2FA accounts.

To test:
- If you don't already have one, create an account using SIWA.
- On the WordPress account in a browser, enabled Two-Step Authentication.
- In the app, try to log in with that account using SIWA.
- Verify you get prompted for a verification code, and you can log in.

<img width="400" alt="verification_prompt" src="https://user-images.githubusercontent.com/1816888/69762380-40fde800-1127-11ea-8c69-53c0f3301d3a.png">

PR submission checklist:
